app/scripts/HorizontalMultivecTrack.js

```diff
@@ -27,7 +27,7 @@ export default class HorizontalMultivecTrack extends HeatmapTiledPixiTrack {
     ctx.fillRect(0, 0, canvas.width, canvas.height);
 
     if (pixData.length) {
-      const pix = new ImageData(pixData, pixData.length / 4, canvas.height);
+      const pix = new ImageData(pixData, canvas.width, canvas.height);
```

Comment:
Could you elaborate in the PR's description what this changes? Seems to be more than just docs updates?

Reply:
This is necessary because some tiles may not be full but the texture should always be the size of the canvas.

Reply:
(This is also the result of a stacked PR)